lib/galaxy/model/migrations/dbscript.py

```diff
@@ -34,6 +34,8 @@
     "22.01": "base",
     "release_22.05": "186d4835587b",
     "22.05": "186d4835587b",
```

Comment:
Should this be backported to 23.0?

Reply:
Yeah, of course! I'll split this into 2 commits: the TS commit will go into dev (no alembic in 23.0)